stream/src/placeholder/handshake.rs

```diff
@@ -18,32 +21,35 @@ fn suffix_namespace(namespace: &[u8]) -> Vec<u8> {
     union(namespace, NAMESPACE_SUFFIX_HANDSHAKE)
 }
 
-pub fn create_handshake<E: Clock, C: Scheme>(
-    runtime: E,
+/// Get the current time in milliseconds since the Unix epoch.
+/// If the system clock is set to before the Unix epoch, this function will panic.
+/// The result saturates at the maximum value of u64.
+pub fn get_current_time_ms<E: Clock> (runtime: &E) -> u64 {
```

Comment:
nit: Given we don't have any other `get_current_time`, I'd recommend `get_current_time` instead.

Reply:
changed to `epoch_time` and returns a duration